minvws/nl-kat-coordination
# First version of kat_politie_check_tradepartner

### Changes

Added a new boefje/normalizer/finding type that uses `https://www.politie.nl/aangifte-of-melding-doen/controleer-handelspartij.html` to see for fraudulent links.

### Demo

![image](https://github.com/user-attachments/assets/9d77ed8c-e195-4afb-b001-556b53fec42e)

### QA notes

Enable this boefje and add some dummy Hostnames to OpenKAT. These hostnames should not be marked unless they are inside of this list:
https://www.politie.nl/aangifte-of-melding-doen/bekende-malafide-handelspartijen.html

### Notes

I am not sure if the finding type should be created inside the normalizer and also be added in the `kat_finding_types.json`.
90% of the work has been done by @underdarknl 

Feedback is appreciated.

---

### Code Checklist







---

## Checklist for code reviewers:


---

## Checklist for QA:


### What works:

- _bullet point + screenshot (if useful) per tested functionality_

### What doesn't work:

- _bullet point + screenshot (if useful) per tested functionality_

### Bug or feature?:

- _bullet point + screenshot (if useful) if it is unclear whether something is a bug or an intended feature._
